project-fob/Controllers/HomeController.cs

```diff
@@ -79,135 +79,109 @@ public ActionResult MeetingPageHost(string attendeePassword, string hostPassword
 
             db.Meeting.Add(meet);
             HttpContext.Session.Set("meetingid", Encoding.ASCII.GetBytes(meet.MeetingId));
-            //Session["meetingid"] = meet.MeetingId;
 
             db.Fob.Add(new Fob(meet));
             db.SaveChanges();
 
             @ViewBag.title = "Meeting Id: " + meet.MeetingId;
-            //TODO generae QR code for the meeting and display it on its own page (add something to host to get this if lost)
+            ViewBag.meetingid = meet.MeetingId;
 
             return View();
         }
 
-
-        //Join Meeting                                          //will this string be empty or null ?
         public ActionResult MeetingPageUser(string meetingId, string password)
-        { //The password will require different way to send it because atm it is visible
+        {
+            if (password == null)
             {
-                if (password == null)
-                {
-                    password = "";
-                }
-                meetingId = meetingId.ToUpper();
-                Meeting meet = db.Meeting.SingleOrDefault(m => m.MeetingId == meetingId);
+                password = "";
+            }
+            meetingId = meetingId.ToUpper();
+            Meeting meet = db.Meeting.SingleOrDefault(m => m.MeetingId == meetingId);
 
-                if (meet != null && meetingId.Equals(meet.MeetingId))
+            if (meet != null && meetingId.Equals(meet.MeetingId))
+            {
+                //host
+                if (password.Equals(meet.HostPassword.ToString()))
```

Comment:
There's no need to `ToString()` something that's already a string, and using `.Equals` for a string isn't needed either. i.e. this could be `if (password == meet.HostPassword)`

Reply:
I removed the ToString(). I use .Equals as it's sometime possible for 2 string (same text, same formating etc.) to be not equal. It's more of a Java thing but a few time it also happened in C# (without changing the string type).

Reply:
That does happen in Java, but in C# `==` on strings compares the value of the `string` (i.e. the character in the string), it's not just checking "is this the same object instance" (which is what happens in Java)